src/terms.jl

```diff
@@ -169,7 +242,7 @@ end
 
 # * Intermediate terms, seniority
 """
-    struct IntermediateTerm
+    IntermediateTerm(term, seniority)
```

Comment:
I am wondering if we should call this something like `SubshellTerm` to make it more clear what it is.

Reply:
I would like to procrastinate on this one.